x/staking/client/cli/cli_test.go

```diff
@@ -1260,6 +1263,75 @@ func (s *IntegrationTestSuite) TestNewCmdUnbond() {
 	}
 }
 
+// TestBlockResults tests that the validator updates correctly show when
+// calling the /block_results RPC endpoint.
+// ref: https://github.com/cosmos/cosmos-sdk/issues/7401.
+func (s *IntegrationTestSuite) TestBlockResults() {
+	val := s.network.Validators[0]
+
+	// Create new account in the keyring.
+	info, _, err := val.ClientCtx.Keyring.NewMnemonic("NewDelegator", keyring.English, sdk.FullFundraiserPath, hd.Secp256k1)
+	s.Require().NoError(err)
+	newAddr := sdk.AccAddress(info.GetPubKey().Address())
+
+	// Send some funds to the new account.
+	_, err = banktestutil.MsgSendExec(
+		val.ClientCtx,
+		val.Address,
+		newAddr,
+		sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(200))), fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+	)
+	s.Require().NoError(err)
+
+	// Use CLI to create a delegation from the new account to validator `val`.
+	s.Require().NoError(err)
+	delHeight, err := s.network.LatestHeight()
+	s.Require().NoError(err)
+	cmd := cli.NewDelegateCmd()
+	_, err = clitestutil.ExecTestCLICmd(val.ClientCtx, cmd, []string{
+		val.ValAddress.String(),
+		sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(150)).String(),
+		fmt.Sprintf("--%s=%s", flags.FlagFrom, newAddr.String()),
+		fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+	})
+	s.Require().NoError(err)
```

Comment:
why not creating a variable in the top of the function:

```
var require = s.Require()
```

Reply:
Same as above, for consistency, would prefer to keep it consistent with the rest of the SDK. In my humble opinion, I actually prefer `s.Require()` in terms of readability (you know you're in a suite).

Reply:
1. If you are in the suite, you shouldn't use the package - so it makes sense to hide the package (because it shouldn't be used at all)
2. It's more concise,  and in few places we use it.

I don't see any argument using a "longer" form, except that it's already used in many places.

Reply:
More concise code usually => easier to read and understand.

Reply:
Again, I prefer `s.Require()` in terms of readability, but sure let's do it your way.

Reply:
thanks. It's easier to read.